VA-Explorer/va_explorer
# Issue140 - added ability to filter through IDs in data management tab

This PR closes issue 140.
Changes: The changes add ability to filter with ID in the data management view. Currently, you can only filter with 1 ID in the field.
Bad inputs are validated.

These changes are useful as they will add another way to filter for rows of VAs.

Testing instructions:
1.) Open data_management tab with mock data installed.
2.) Try filtering and rows based on ID value. You can also check your mock data file to make sure ID is accurate. Make sure the view updates the rows of VAs for the filter items.
3.) For validation, try entering bad input. Such as "123abc" or "12$" or "5b", you can also enter negative numbers such as "-4" which are also validated out.
